changelogs/fragments/69352-netbsd-virtual-facts.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
+  - NetBSD virtualization facts (specifically `ansible_virtualization_type`) now returns a more accurate value by checking the value of the `machdep.hypervisor` `sysctl` key. This change is breaking because in some cases previously, we would erroneously report `xen` even when the target is not running on Xen. This prevents that behavior in most cases.
```

Comment:
```suggestion
  - NetBSD virtualization facts (specifically ``ansible_virtualization_type``) now returns a more accurate value by checking the value of the ``machdep.hypervisor`` ``sysctl`` key. This change is breaking because in some cases previously, we would erroneously report ``xen`` even when the target is not running on Xen. This prevents that behavior in most cases. (https://github.com/ansible/ansible/issues/69352)
```